.changeset/cuddly-yaks-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@atproto/oauth-client": minor
+---
+
+Use `private_key_jwt` instead of incorrect `client_secret_jwt` as authentication method for confidential clients
```

Comment:
I don't think `client_secret_jwt` was ever listed as a valid authentication method for confidential clients, at least not in a version that landed in `main` or was published.